account_invoice_report_hls/report/invoice_delivery.xml

```diff
@@ -0,0 +1,145 @@
+                        <span t-field="o.invoice_id.partner_id.zip"/><br/>
+                      </t>
+                      <t t-if="o.invoice_id.partner_id.state_id">
+                        <span t-field="o.invoice_id.partner_id.state_id.name"/>
+                      </t>
+                      <t t-if="o.invoice_id.partner_id.city">
+                        <span t-field="o.invoice_id.partner_id.city"/>
+                      </t>
+                      <t t-if="o.invoice_id.partner_id.street">
+                        <span t-field="o.invoice_id.partner_id.street"/>
+                      </t>
+                      <t t-if="o.invoice_id.partner_id.street2">
+                        <span t-field="o.invoice_id.partner_id.street2"/>
+                      </t>
+                      <br/>
+                      <br/>
+                      <span t-field="o.invoice_id.partner_id"/>
+                    </div>
+                    <div style="padding-top:50px;">
+                      <span>平素は格別のご高配に賜り厚くお礼申し上げます。下記の通り、ご請求申し上げます。</span>
+                    </div>
+                  </div>
+                  <div class="col-6 float-right">
+                    <div style="background-color: #e0e0eb;text-align:center;padding-bottom:5px;padding-top:10px;">
+                      <h4>御取引明細書兼請求書 <span t-field="o.invoice_id.doc_title"/></h4>
+                    </div>
+                    <div style="padding-top:30px;">
+                      <strong><span t-field="o.invoice_id.company_id"/></strong><br/>
+                        <t t-if="o.invoice_id.company_id.zip">
+                          <span t-field="o.invoice_id.company_id.zip"/><br/>
+                        </t>
+                        <t t-if="o.invoice_id.company_id.state_id">
+                          <span t-field="o.invoice_id.company_id.state_id.name"/>
+                        </t>
+                        <t t-if="o.invoice_id.company_id.city">
+                          <span t-field="o.invoice_id.company_id.city"/>
+                        </t>
+                        <t t-if="o.invoice_id.company_id.street">
+                          <span t-field="o.invoice_id.company_id.street"/>
+                        </t>
+                        <t t-if="o.invoice_id.company_id.street2">
+                          <span t-field="o.invoice_id.company_id.street2"/>
+                        </t>
+                        <t t-if="o.invoice_id.company_id.phone">
+                          <br/><span>Tel: </span><span t-field="o.invoice_id.company_id.phone"/> 
+                        </t>
+                        <t t-if="o.invoice_id.company_id.fax">
+                          <span> Fax: </span><span t-field="o.invoice_id.company_id.fax"/> 
+                        </t>
+                        
+                    </div>
+                    <br/>
+                    <br/>
+                    <table style="border:1px solid black;width:100%">
```

Comment:
@AungKoKoLin1997 Can you try and minimize the repetitive style assignments (which is ugly)? Use available classes where applicable.